GameData/ROEngines/PartConfigs/GEM63_BDB.cfg

```diff
@@ -0,0 +1,94 @@
+PART
+{
+	module = Part
+	name = ROE-GEM63
+	author = Cobaltwolf, Vader111
+
+	RSSROConfig = true
+	ROESetEngineDefaults = SOLID_BOOSTER
+
+	mass = 5.240
+
+	//	============================================================================
+	//	Update Below
+	//	============================================================================
+
+	MODEL
+	{
+		model = ROEngines/Assets/BDB/Delta/bluedog_Delta_GEM63
+	}
+
+	scale = 1.0
+	rescaleFactor = 1.6
+	node_attach = 0.0, 0.0, -0.46875, 0.0, 0.0, 1.0, 1
+	CoMOffset = 0.0, -0.235, -0.235
+	// stack, srfAttach, allowStack, allowSrfAttach, allowCollision
+	attachRules = 1,1,1,1,0	
+
+	title = GEM 63
+	manufacturer = Orbital ATK
+	description = In 2015, ULA announced that Orbital ATK had been contracted to develop the GEM 63 as a drop-in replacement for the AJ-60A, and would replace Aerojet Rocketdyne as their Atlas V SRM supplier by the end of 2018. The GEM 63 will be larger than the GEM 60 SRMs used on the Delta IV and will not be equipped with thrust vectoring. As with the AJ-60A, launches using the four-meter fairing will be able to be equipped with up to three boosters while up to five boosters can be used with the five-meter fairing. 
+
+	tags = ATK, GEM63, booster, radial, solid, srb,
+
+	engineType = GEM-63
+
+	skinInternalConductionMult = 4.0
+	emissiveConstant = 0.5
+	dragModelType = default
+	maximum_drag = 0.3
+	minimum_drag = 0.2
+	angularDrag = 2
+
+	MODULE
+	{
+		name = ModuleEnginesRF
+		thrustVectorTransformName = thrustTransform
+	}
+
+	MODULE
+	{
+		name = ModuleB9PartSwitch
+		moduleID = meshSwitchNose
+		switcherDescription = Nose Shape
+		switcherDescriptionPlural = Nose Shapes
+		affectDragCubes = True
+		affectFARVoxels = True
+		SUBTYPE
+		{
+			name = Straight
+			transform = StraightNose
+		}
+		SUBTYPE
+		{
+			name = Angled
+			transform = AngledNose
+		}
+	}
+
+	MODULE
+	{
+		name = ModuleSurfaceFX
+		thrustProviderModuleIndex = 0
+		fxMax = 1
+		maxDistance = 80
+		falloff = 2
+		thrustTransformName = thrustTransform
+	}
+
+	MODULE
+	{
+		name = FXModuleAnimateThrottle
+		animationName = GEM60_Emit_Anim
+		dependOnEngineState = True
+		responseSpeed = 0.001
+	}
+}
+
+@PART[ROE-GEM63]:AFTER[RealismOverhaulEnginesPost] 
+{
+	@MODULE[ModuleEngineConfigs] 
+	{ 
+		@origMass = 5.240
+	}
+}
```

Comment:
If the mass values in the RO engine config are incorrect, please PR the fix there